servicebus/spring-cloud-azure-stream-binder-servicebus/servicebus-topic-binder/src/main/resources/application-sp.yaml

```diff
@@ -2,10 +2,10 @@ spring:
   cloud:
     azure:
       credential:
-        client-id: [ client-id ]
-        client-secret: [ client-secret ]
+        client-id: ${AZURE_CLIENT_ID}
+        client-secret: ${AZURE_CLIENT_SECRET}
       profile:
-        tenant-id: [ tenant-id ]
+        tenant-id: ${AZURE_TENANT_ID}
 
 #    Uncomment below configurations if you want to enable auto creating resources.
```

Comment:
Please remove the configuration of auto-creating-resources in all the configuration files and readme.